.gitignore

```diff
@@ -25,3 +25,6 @@
 .project
 .envrc
 /inventory.yaml
+/vagrant/modules/
+/vagrant/Puppetfile.lock
+/ubuntu-bionic-18.04-cloudimg-console.log
```

Comment:
I would scrap the "/" from from of this line. I recall that this log file gets created in the directory where you run "vagrant up" from, and this only matches if the file is at the root of the repository.